projects/Mallard/src/components/article/article-kicker/index.tsx

```diff
@@ -2,28 +2,28 @@ import React from 'react'
 import { View } from 'react-native'
 import { useArticleAppearance } from 'src/theme/appearance'
 import { HeadlineKickerText } from 'src/components/styled-text'
-import { longReadHeaderStyles, newsHeaderStyles } from './article-header-styles'
+import { longReadHeaderStyles, newsHeaderStyles } from '../styles'
 
-const dontDisplayKicker = ['Opinion']
+const dontDisplayKicker = ['opinion']
```

Comment:
This seems redundant now we have an opinion header, couldn't we just not use this component in the `OpinionHeader`?

Reply:
Yep - good shout. Changed!